src/medkit/medkit.tsx

```diff
@@ -10,6 +12,28 @@ interface MedkitItem {
     points: string;
 }
 
+const resetMedkitData = (): void => {
+    const initList = (new Array<boolean>(DATA.length)).fill(false);
+    window.localStorage.setItem('medkit', JSON.stringify(initList));
+};
+
+export const initMedkitData = (): void => {
+    if (window.localStorage.getItem('medkit')) {
+        return;
+    }
+    resetMedkitData();
+};
+
+export const getMedkitData = (): boolean[] => {
+    return JSON.parse(window.localStorage.getItem('medkit')) as boolean[];
+};
+
+export const setMedkitData = (idx: number, value: boolean): void => {
+    const data = JSON.parse(window.localStorage.getItem('medkit')) as boolean[];
```

Comment:
Verified this is working as expected. I don't see where the data is loading into the interactive from localStorage. Is that still todo?

Reply:
That's still todo, I'll have a PR by the end of the day.

Reply:
Great. Merging this.